poll/public/css/poll.css

```diff
@@ -145,7 +145,7 @@ li.poll-result .poll-image {
 
 .survey-table thead tr th {
     font-weight: bold;
-    font-size: .8rem;
+    font-size: .9rem;
```

Comment:
The QA folks requested that the answer text be bold. It was already bold, so I'm guessing that they just wanted it a bit larger. Please let me know if this is in error.